go.mod

```diff
@@ -117,9 +117,9 @@ require (
 	github.com/pkg/errors v0.8.0
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/pquerna/ffjson v0.0.0-20180717144149-af8b230fcd20 // indirect
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/client_model v0.0.0-20180712105110-5c3871d89910
-	github.com/prometheus/common v0.0.0-20181126121408-4724e9255275
+	github.com/prometheus/client_golang v1.0.0
```

Comment:
I'm noticing the `client_golang` is going from 0.9.2 to 1.0.0. If its following semver, it would appear that major version bump has the potential for breaking changes. Have we taken a look to see if there are any unexpected changes that impact us?